src/mixins/filing-template-mixin.ts

```diff
@@ -400,7 +400,7 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
     this.setRegistrationBusinessAddress(draftFiling.registration.offices?.businessOffice)
 
     // restore Business Type
-    this.setRegistrationBusinessType(draftFiling.registration.businessType || BusinessTypes.SP)
+    this.setRegistrationBusinessType(draftFiling.registration.businessType)
```

Comment:
For the reviewers: I asked for this change. It is so that Business Type is not defaulted to SP for General Partnership. I confirmed with @vysakh-menon-aot that Business Type does not apply to GPs.